core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -852,6 +846,7 @@ paths:
             - profile
   /api/admin/statistics:
     get:
+      deprecated: true
```

Comment:
this endpoint is slated for removal here: https://github.com/broadinstitute/rawls/pull/1342